examples/iPi/zundel/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# remove the driver file if it already exists
+rm /tmp/ipi_zundel
+# make sure rascal can be imported if not installed
+export PYTHONPATH="../../../build_b/:$PYTHONPATH"
```

Comment:
This will not run as-is on most people computers, since it is refering to files outside of the librascal repository. Could you replace/improve this file with a README explaining all that is required to run this example?

Reply:
Or even just replace this line and the  RASCAL_DRIVER  below with `PYTHONPATH="<path/to/rascal/install>:$PYTHONPATH"` and in the README tell people they need to edit this file before running anything.